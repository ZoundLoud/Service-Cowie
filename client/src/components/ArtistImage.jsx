import React from 'react';

const ArtistImage = props => (
  <span className="imageContainer">
    <img className="image" src="https://i1.wp.com/thissongissick.com/wp-content/uploads/2018/10/ed-sheeran-and-passenger-no-diggity-vs-thrift-shop-kygo-remix-must-hear-chill-house-free-download.jpg?w=640&quality=88&strip&ssl=1" alt="" />
  </span>
);

export default ArtistImage;
