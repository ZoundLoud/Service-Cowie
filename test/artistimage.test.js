import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { configure, mount, shallow } from 'enzyme';
import ArtistImage from '../client/src/components/ArtistImage';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

describe('ArtistImage Component', () => {
  it('should render correctly according to snapshot', () => {
    const props = {
      imgurl: 'https://s3-us-west-1.amazonaws.com/songimages/photos/simg90.jpg',
    };
    const artistimage = shallow(<ArtistImage img={props.imgurl} />);
    expect(toJson(artistimage)).toMatchSnapshot();
  });
  it('should render correctly in "debug" mode', () => {
    const props = {
      imgurl: 'https://s3-us-west-1.amazonaws.com/songimages/photos/simg90.jpg',
    };
    const debug = shallow(<ArtistImage img={props.imgurl} debug />);
  
    expect(toJson(debug)).toMatchSnapshot();
  });
});
