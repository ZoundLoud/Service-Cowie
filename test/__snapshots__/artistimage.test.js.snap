// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistImage Component should render correctly according to snapshot 1`] = `
<span
  className="imageContainer"
>
  <img
    alt=""
    className="image"
  />
</span>
`;
